README.md

```diff
@@ -107,6 +107,20 @@ EOF
 systemctl enable scanpi --now
 ```
 
+## Docker Compose Template
+```
+services:
+  scanpi:
+    container_name: scanpi
+    image: # TODO: Add image here
```

Comment:
For here, you can use `build: .` to have docker-compose build from the docker file. I will be adding a CICD to publish the container once I have some other issues worked through.